assets/js/components/flows/FlowsWorkspace.jsx

```diff
@@ -231,18 +238,22 @@ export default ({
             selectNodesOnDrag={false}
             multiSelectionKeyCode={null}
             selectionKeyCode={null}
+            nodesDraggable={userCan({ role: currentRole })}
```

Comment:
are these still used?

Reply:
this one, yes, since it prevents readonly user from dragging the nodes around, the 2nd one maybe redundant since I'm checking in the add+remove functions so I can take that one away

Reply:
the 2nd one is also being used actually to prevent the edge from forming when you try to drag (even though it won't connect bc of the protection on the add function)
![image](https://user-images.githubusercontent.com/51131939/118698565-cc60b280-b7cd-11eb-8c69-b46fd4d3ce11.png)

Reply:
got it, thanks!